prospector.yml

```diff
@@ -35,6 +35,15 @@ pep257:
     - D105
     - D211
     - D104
+    - D212  # Multi-line docstring summary should start at the first line
+    - D107  # Missing docstring in __init__
```

Comment:
We usually put the docstring at the class level.

(I personally prefer in the `__init__`, but... it just a preference)